tests/Functional/Action/LtiInstance/CreateActionTest.php

```diff
@@ -77,7 +77,7 @@ public function testValidRequest(): void
             [],
             [],
             [],
-            json_encode([
+            (string)json_encode([
```

Comment:
Do we really need to type cast here? Is there any chance to get something other than string?

Reply:
well. yes :)
```
Returns a JSON encoded string on success or false on failure.
```

Reply:
It's for static analyser. `json_encode` could return bool value but in tests we don't care.